Containerfile

```diff
@@ -0,0 +1,8 @@
+FROM docker.io/peaceiris/mdbook:v0.4.15
+
+RUN apk update && apk add hugo
```

Comment:
It seems like we even could use the script to launch it inside a `nix` container.

Reply:
What exactly do you mean? Launching the container from inside a nix environment?

Reply:
It was mostly a joke, but:

Using the `nixos/nix` container we used to test the command.
That way we would use the script to start the container, and then run the script again inside the container and install and run the dependencies.